torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3955,6 +3955,48 @@ def sample_inputs_matmul(op_info, device, dtype, requires_grad):
     return tuple(sample_inputs)
 
 
+def sample_inputs_meshgrid(
+        calling_convention: str) -> Callable[[OpInfo, torch.device, torch.dtype, bool], List[SampleInput]]:
```

Comment:
Adding all this type information looks time-consuming

Reply:
I don't remember if it made me do it. It seems unlikely given that most of the file is not type annotated.